src/styles/card.scss

```diff
@@ -0,0 +1,104 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  box-sizing: border-box;
+  max-width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: $borderRadius;
+  padding-bottom: 16px;
+
+  background-color: #fff;
+
+  &__top {
+    &-image {
+      width: 160px;
+      height: 134px;
+      margin-left: 3px;
+      margin-top: 32px;
+    }
+
+    &-title {
+      max-width: $blockWidth;
+      margin-top: 36px;
+
+      font-size: 12px;
+      font-weight: 500;
+      line-height: 18px;
+      color: $titleTextColor;
+    }
+
+    &-product-code {
+      margin-top: 4px;
+
+      font-size: 10px;
+      font-weight: 400;
+      line-height: 14px;
+      color: $underTitleTextColor;
+    }
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    max-width: $blockWidth;
+
+    margin-top: 16px;
+
+    &-count {
+      font-size: 10px;
+      font-weight: 400;
+      line-height: 14px;
+      color: $titleTextColor;
+      margin-left: 14px;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    max-width: $blockWidth;
+    margin-top: 24px;
+
+    &-text {
+      font-size: 12px;
+      font-weight: 400;
+      line-height: 18px;
+      color: $underTitleTextColor;
+      margin-right: 86px;
+    }
+
+    &-count {
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+      color: $titleTextColor;
+    }
+  }
+
+  &__button-buy {
+    display: flex;
+    box-sizing: border-box;
+    justify-content: center;
+    align-items: center;
+    max-width: $blockWidth;
+    max-height: 40px;
+    padding: 18px 80px;
+    margin-top: 16px;
+    border-radius: $borderRadius;
+
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    color: #fff;
+    text-decoration: none;
+    background-color: $buttonTextColor;
+
+    &:hover {
+      border: 1px solid $buttonTextColor;
```

Comment:
The color of hovered button border is the same as a normal button background, so you can set the border to the button don't change it on hover

the less styles you alter the better